lib/aerospike/aerospike_exception.rb

```diff
@@ -65,6 +94,10 @@ class Connection < Aerospike
       def initialize(msg=nil)
         super(ResultCode::SERVER_NOT_AVAILABLE, msg)
       end
+
+      def retryable?
```

Comment:
Do we have to overwrite it? We handle `SERVER_NOT_AVAILABLE` here`:

https://github.com/castle/aerospike-client-ruby/pull/5/files#diff-fc56053424973707d0ca764bb54a373164030ab4ec934de00f76a39dbec6654fR46